src/main/java/com/netgrif/application/engine/petrinet/service/PetriNetService.java

```diff
@@ -110,32 +112,40 @@ public class PetriNetService implements IPetriNetService {
     @Autowired
     private IUriService uriService;
 
-    private Map<ObjectId, PetriNet> cache = new HashMap<>();
+    @Autowired
+    private CacheManager cacheManager;
 
     protected Importer getImporter() {
         return importerProvider.get();
     }
 
     @Override
     public void evictCache() {
-        cache = new HashMap<>();
+        cacheManager.getCache("netsId").clear();
```

Comment:
Names of caches should be from app properties not as hardcoded strings.

Reply:
done